projects/plugins/protect/CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.1.0 - 2022-11-17
+### Added
+- Added features for paid Jetpack Scan users; including file and database threats, on-demand scanning, and threat auto-fixers.
```

Comment:
Generally we try to keep with present tense for the changelog files - not too worried about the other package updates, but moreso for the plugin changelog itself since that's what's most user facing.

```suggestion
- Add features for paid Jetpack Scan users, including file and database threats, on-demand scanning, and threat auto-fixers.
```
> including file and database threats

Is this for file and database threat scanning? Or are we including the threats themselves in the update? :trollface: